docker-compose.yml

```diff
@@ -107,6 +107,7 @@ services:
       GATEWAY_SERVER_URL: http://aerie_gateway:9000
       HASURA_CLIENT_URL: http://localhost:8080/v1/graphql
       HASURA_SERVER_URL: http://hasura:8080/v1/graphql
+      HASURA_WEB_SOCKET_URL: ws://localhost:8080/v1/graphql
```

Comment:
Nevermind, just read the README:
>  `HASURA_CLIENT_URL`     | Url of Hasura as called from the client (i.e. web browser)                                                | `string` | http://localhost:8080/v1/graphql |
| `HASURA_SERVER_URL`     | Url of Hasura as called from the server (i.e. Node.js container)

Reply:
The `HASURA_SERVER_URL` is using the hostname of the hasura Docker container since that is called during server-side rendering. The `HASURA_WEB_SOCKET_URL` and `HASURA_CLIENT_URL` are called from the browser so use `localhost`.